packages/calcite-components/src/components/carousel/carousel.tsx

```diff
@@ -528,7 +566,8 @@ export class Carousel
   };
 
   private tabListKeyDownHandler = (event: KeyboardEvent): void => {
-    const interactiveItems = Array(...this.tabList.querySelectorAll("button"));
+    const targetEls = `button:not(.${CSS.paginationItemOutOfRange})`;
```

Comment:
Can you name this variable to indicate both what it's targeting and that it's a selector? This can also be inlined below.

Reply:
Updated!